src/framework/mlt_frame.c

```diff
@@ -445,6 +470,9 @@ int mlt_image_format_size( mlt_image_format format, int width, int height, int *
 		case mlt_image_glsl_texture:
 			if ( bpp ) *bpp = 0;
 			return 4;
+		case mlt_image_yuv422p16:
+			if ( bpp ) *bpp = 2;
```

Comment:
The bpp param is not useful for computing stride for planar formats. Any code that is calling this to get bpp that can work with mlt_image_yuv422p16 is not going to use this param, and it does not matter. Might be safer to set bpp to zero to prevent it from being used for this image format.